src/app/file-manager/file-download-modal/file-download-modal.component.ts

```diff
@@ -74,25 +74,28 @@ export class FileDownloadModalComponent  {
   }
 
   setItems() {
-    this.data.selectedFiles.filter(file => file.data.type === 'File')
+    this.data.selectedFiles.filter(file => file.type === 'File')
       .forEach(file => {
         const dataFile = {
-          id: file.data.id,
-          name: file.data.name,
-          size: file.data.size,
-          created: file.data.updated,
-          updated: file.data.updated,
-          icon: file.data.type === 'Folder' ? 'folder' : 'cloud',
+          id: file.id,
+          name: file.name,
+          size: file.size,
+          created: file.updated,
+          updated: file.updated,
+          icon: file.type === 'Folder' ? 'folder' : 'cloud',
```

Comment:
It looks like `Folder` will never occur because of the call to `.filter()` above.